projects/plugins/jetpack/modules/custom-css/custom-css.php

```diff
@@ -690,24 +762,26 @@ static function link_tag() {
 			return;
 		}
 
-		if ( ! is_super_admin() && isset( $current_blog ) && ( 1 == $current_blog->spam || 1 == $current_blog->deleted ) )
+		if ( ! is_super_admin() && isset( $current_blog ) && ( 1 == $current_blog->spam || 1 == $current_blog->deleted ) ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
```

Comment:
The fields are documented as being "numeric strings", but I wonder if we could just do like
```suggestion
		if ( ! is_super_admin() && isset( $current_blog ) && ( $current_blog->spam || $current_blog->deleted ) ) {
```
i.e. are they really supposed to be bools?

Reply:
Could give it a go.